docs/pages/divider.js

```diff
@@ -1,64 +1,309 @@
+            defaultCode={`
+function Example() {
+  const Block = ({ title, text }) => (
+    <Flex direction="column" gap={2}>
+      <Heading size="sm">{title}</Heading>
+      <Text size="md">{text}</Text>
+    </Flex>
+  );
+
+  return (
+    <Flex direction="column" gap={10}>
+      <Block title="Discover ideas" text="Use the searh bar to discover ideas, people and trends. Explore suggested topics or search for topics of your own." />
+      <Flex width="100%" justifyContent="center">
+        <Box width="50%">
+          <Divider />
+        </Box>
+      </Flex>
+      <Block title="Create Pins" text="Upload an image from your computer or mobile device to create a Pin. You can also create Pins from images yu find online." />
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use Dividers to separate collections of content or create the appearance of containers."
+            defaultCode={`
+function Example() {
+  const Block = ({ title, text }) => (
+    <Flex direction="column" gap={2}>
+      <Heading size="sm">{title}</Heading>
+      <Text size="md">{text}</Text>
+    </Flex>
+  );
+
+  return (
+    <Flex>
+      <Box paddingX={4}>
+        <Block title="Discover ideas" text="Use the searh bar to discover ideas, people and trends. Explore suggested topics or search for topics of your own." />
+      </Box>
+      <Divider />
+      <Box paddingX={4}>
+        <Block title="Create Pins" text="Upload an image from your computer or mobile device to create a Pin. You can also create Pins from images yu find online." />
+      </Box>
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use Dividers for decoration or to separate grouped content"
+            defaultCode={`
+function Example() {
+  return (
+    <Flex width="80%">
+      <Box paddingX={4}>
+        <Heading size="sm">Create Pins</Heading>
+      </Box>
+      <Divider />
+      <Box paddingX={4}>
+        <Text size="md">Upload an image from your computer or mobile device to create a Pin. You can also create Pins from images yu find online.</Text>
+      </Box>
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Accessibility">
+        <MainSection.Subsection description="The Divider is not focusable and is treated as decorative. Screen readers on tab navigation don't announce Dividers but do announce them on left/right quick navigation." />
+      </MainSection>
+      <MainSection name="Localization">
+        <MainSection.Subsection description="If you are aligning the Divider to content, ensure that it switches sides according to the contents start-alignment" />
```

Comment:
"…according to the content's start-end alignment."
- apostrophe on "content"
- "start-end" to clarify which alignment we're referring to
- period at the end